anitya/lib/utilities.py

```diff
@@ -67,13 +68,27 @@ def check_project_release(project, session, test=False):
     versions_prefix = []
     max_version = ''
 
+    # don't change actual data during test run
+    if not test:
+        project.last_check = datetime.now()
+        project.next_check = project.last_check + backend.check_interval
+
     try:
         versions_prefix = backend.get_versions(project)
     except exceptions.AnityaPluginException as err:
         _log.exception("AnityaError catched:")
-        project.logs = str(err)
-        session.add(project)
-        session.commit()
+        if not test:
+            project.logs = str(err)
+            session.add(project)
+            session.commit()
+        raise
+    except exceptions.RateLimitException as err:
+        _log.exception("AnityaError catched:")
+        if not test:
+            project.logs = str(err)
+            project.next_check = err.reset_time.to('utc').naive
```

Comment:
You probably don't want the naive time here

Reply:
I found out, that I need this for python 2.7. Otherwise tests are starting to fail.

Reply:
Why are you trying to support Python 2.7? I don't see any value in it, I'd say support 3.6+ or something.

Reply:
You think it is time to drop 2.7 tests from tox and CI?

Reply:
But I drop them with pleasure

Reply:
But first I update this PR